op-service/sources/batching/test/event_stub.go

```diff
@@ -0,0 +1,63 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-service/sources/batching/rpcblock"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+type expectedFilterLogsCall struct {
+	topics  [][]common.Hash
+	to      common.Address
+	block   rpcblock.Block
+	outputs []types.Log
+	err     error
+}
+
+func (c *expectedFilterLogsCall) Matches(rpcMethod string, args ...interface{}) error {
+	if rpcMethod != "eth_getFilterLogs" {
+		return fmt.Errorf("expected rpcMethod eth_getFilterLogs but was %v", rpcMethod)
+	}
+
+	topics := args[0].([][]common.Hash)
```

Comment:
This will panic if `args[0]` is not the expected type, is this the expected behavior?